nav2_behavior_tree/test/plugins/condition/test_goal_reached.cpp

```diff
@@ -45,7 +45,7 @@ class GoalReachedConditionTestFixture : public nav2_behavior_tree::BehaviorTreeT
       R"(
       <root main_tree_to_execute = "MainTree" >
         <BehaviorTree ID="MainTree">
-            <GoalReached goal="{goal}" />
+            <GoalReached goal="{goal}" global_frame="map" robot_base_frame="base_link"/>
```

Comment:
Should this be strictly required if its in the declared parent parameters?

Reply:
I think I need to either add it as a ROS2 parameter, or directly in the BT tree for this test to work correctly.